frontend/Views/NewDiscussion/actions.js

```diff
@@ -60,14 +62,6 @@ export const postDiscussion = (userId, forumId, currentForum) => {
       });
     }
 
-    if (tags === null || tags.length === 0) {
```

Comment:
Instead of straight up removing this code, can we refactor this to use a constant like MINIMUM_TAG_LENGTH, like what you added with the title characters? thanks!

Reply:
Done!